functions/src/services/validate-request.ts

```diff
@@ -0,0 +1,40 @@
+import { body, validationResult } from "express-validator";
+import { NextFunction, Request, Response } from "express";
+
+/**
+ *
+ * Function to enforce a required field in request body
+ *
+ * @param fieldName - name of field that needs to be enforced as required
+ * @param onErrorMsg - if this field is not included, an error object will be returned including the message stored in this variable
+ *
+ *
```

Comment:
```suggestion
```

Reply:
Additionally, make sure to add a `@return` doc comment to make it clear what's being returned.